Sources/ScipioKit/SwiftPM/Version.swift

```diff
@@ -0,0 +1,300 @@
+// swiftlint:disable:next blanket_disable_command
+// swiftlint:disable all
```

Comment:
We should add `excluded` entry to `.swiftlint.yml` for this directory instead of adding disable comments.

https://github.com/giginet/Scipio/blob/c96f2605ca958d7fdae80bb1a892918049d023e9/.swiftlint.yml#L4-L6